styles.css

```diff
@@ -647,10 +664,240 @@ form textarea ::placeholder {
 @media only screen and (min-width: 768px) {
-
-}
\ No newline at end of file
+  .page-1_container {
+    margin: auto;
+  }
+
+  header {
+    margin-bottom: -5%;
+    margin-top: -5%;
+  }
+
+  .my-logo {
+    margin-right: 70%;
+    margin-top: 6.5%;
+  }
+
+  .collapsed_menu {
+    display: none;
+  }
+
+  .menu {
+    margin-top: 3%;
+  }
+
+  .headline {
+    width: 100%;
+    background-image: url('images/Header\ bg.png');
+    background-position: fixed;
+    background-size: 100% 80%;
+    padding-bottom: 8%;
+  }
+
+  .headline h1 {
+    margin-left: 17%;
+    margin-bottom: 7%;
+    margin-top: 10%;
+    padding-top: 15%;
+  }
+
+  .headline p {
+    width: 50%;
+    margin-left: 17%;
+  }
+
+  .headline div {
+    margin-left: 17%;
+    margin-top: -3%;
+  }
+
+  .headline ul {
+    margin-left: 17%;
+    margin-top: -2%;
+    margin-bottom: 10%;
+    gap: 2%;
+  }
+
+  .card_works {
+    box-sizing: border-box;
+    display: grid;
+    width: 1156px;
+    height: 476px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 3%;
+  }
+
+  .snapshot_portfolio {
+    float: left;
+    width: 544px;
+    height: 448px;
+    background-size: contain;
+    padding: 0;
+    margin: 0;
+  }
+
+  .snapshot_portfolio_1 {
+    float: right;
+    width: 544px;
+    height: 448px;
+    background-size: contain;
+    padding: 0;
+    margin: 0;
+  }
+
+  .left_block {
+    width: 50%;
+    margin-left: 54%;
+    margin-top: -85%;
+  }
+
+  .work_1 .snapshot_portfolio {
+    background: url('images/Snapshoot\ Portfolio.png');
+    margin-top: -4%;
+    margin-left: -4%;
+  }
+
+  .work_2 .snapshot_portfolio_1 {
+    margin-left: 50%;
+  }
+
+  .work_2 .left_block {
+    margin-left: -2%;
+  }
+
+  .work_3 .snapshot_portfolio {
+    margin-left: -4%;
+    margin-top: -3%;
+  }
+
+  .work_4 .snapshot_portfolio_1 {
+    margin-left: 50%;
+    background: url('images/uber.png');
+    margin-top: -4%;
+  }
+
+  .work_4 .left_block {
+    margin-left: -2%;
+  }
+
+  .card_works + .card_works {
+    margin-top: 10%;
+  }
+
+  .works {
+    margin-left: 10%;
+  }
+
+  .about_header {
+    margin-bottom: -0.1%;
+  }
+
+  .about_description {
+    margin-left: -58%;
+    margin-top: 0.99%;
+    width: 447px;
+  }
+
+  .connect {
+    margin-top: -2%;
+    margin-left: 14%;
+  }
+
+  .about_myself ul {
+    gap: 1%;
+    margin-top: 1%;
+    margin-left: 1%;
+  }
+
+  .resume_button {
+    width: 13%;
+    margin-left: -73%;
+    margin-top: 1%;
+    padding: 0.2%;
+  }
+
+  .languages {
+    margin-left: 87%;
+    margin-top: -17%;
+  }
+
+  .language_header p {
+    margin-right: 500px;
+  }
+
+  .language_list {
+    display: flex;
+    flex-direction: row;
+    width: 20%;
+  }
+
+  .language_list + .language_list {
+    margin-right: 10%;
+  }
+
+  .language_list div {
+    flex-direction: column;
+    margin-right: 10%;
+    margin-bottom: 20%;
+    background-color: f7f7f9;
+    padding-bottom: 30%;
+  }
```

Comment:
- [ ] The `space-between` your languages is  not the same as Figma design kindly try to match your design to the Figma one, this will make your project looks more beautiful and so attractive  :wink: 
Your Design 
![Screenshot from 2023-02-25 18-01-28](https://user-images.githubusercontent.com/88894525/221366998-d26f8992-c24b-4923-9fc0-1ead17676c76.png)


Figma Design 
![Screenshot from 2023-02-25 17-59-20](https://user-images.githubusercontent.com/88894525/221366893-2cb9c5a5-df45-484a-95bc-df85231498aa.png)